### DISCUSSION TITLE
Whatever happened to Harden Windows 10 for Security?

### DISCUSSION THREAD

**Discussion by Logomachist** [HotCakeX/Harden-Windows-Security#217] [General]
Are you guys (or, rather were you guys) familiar with Harden Windows 10 for Security (http://www.hardenwindows10forsecurity.com)?  I saw it some time back and it looked legit but revisiting it now it appears to be defunct, with the domain resolving to some cybersquatter at GoDaddy.  I'm wondering if you guys thought it was any good, what happened to it, and whether this (or some other) site is a worthy replacement.

Anyone know?

**Comment by HotCakeX** [comment]
Hi,
There are some snapshots of it [here](https://web.archive.org/web/*/http://www.hardenwindows10forsecurity.com/). I never analyzed it but i think there was a paid product being offered on that page. IMO that looks like a domain name that could be used for phishing too.

As a replacement, you can use the content on this repository for free, forever.

**Reply by Logomachist** [reply]
There was a paid script that automated some things but mostly it just sped up what otherwise you could do manually, following the instructions on the page.  It wasn't expensive, and I did pay for a copy and the time savings could be worth it IMO.  [From what I gathered](https://www.bleepingcomputer.com/forums/t/690698/hardening-vista-are-these-utilities-all-legit/), the advice was sound.

I'm taking a look at your site now, and I'm curious to see how it compares.

**Reply by HotCakeX** [reply]
I have a video showing how to use it
https://www.youtube.com/watch?v=AksQ0NACRxY&feature=youtu.be

The readme page includes all the details
https://github.com/HotCakeX/Harden-Windows-Security

This is how the module works
https://github.com/HotCakeX/Harden-Windows-Security/wiki/Harden%E2%80%90Windows%E2%80%90Security%E2%80%90Module

Additional info
https://github.com/HotCakeX/Harden-Windows-Security/blob/main/Rationale.md

Basic FAQ
https://github.com/HotCakeX/Harden-Windows-Security/wiki/Answers-to-the-Basic-Frequently-Asked-Questions

Afte reviewing the contend i shared above, if you still have specific questions i'll answer
